packages/utils/src/format.ts

```diff
@@ -27,3 +28,24 @@ export function truncBytes(root: Uint8Array | string): string {
   const str = typeof root === "string" ? root : toHexString(root);
   return str.slice(0, 14);
 }
+
+/**
+ * Formats an address according to [ERC55](https://eips.ethereum.org/EIPS/eip-55)
+ *
+ * @param address an hex address
+ * @returns an ERC55 formatted version of `address`
+ */
+export function toChecksumAddress(address: string): string {
```

Comment:
this tsdoc doesn't really give you much more information as the params are simple enough

```suggestion
 */
export function toChecksumAddress(address: string): string {
```

Reply:
I fell like the mention of ERC55 and associated link is useful

Reply:
I am just talking about the @params and @returns, see diff in code suggestion